corehq/messaging/scheduling/static/scheduling/js/conditional_alert_list.js

```diff
@@ -70,10 +70,11 @@ hqDefine("scheduling/js/conditional_alert_list", function() {
                     "targets": [4],
                     "render": function(data, type, row) {
                         var id = row[row.length - 1];
+                        var editable = row[row.length - 4];
```

Comment:
The indexing here (and in broadcasts_list.js) is getting hard to follow. What would you think of changing the ajax response to return a dictionary instead of a list?

Reply:
agree, i thought that was a limitation on datatables but i guess it's not: 
https://github.com/dimagi/commcare-hq/pull/19217/commits/ea139f6587f2ce99ee1909af129d91f1011ed5f3
https://github.com/dimagi/commcare-hq/pull/19217/commits/79e070c0aab9d3694759640915a39d352cef76f7